README.md

```diff
@@ -1,34 +1,89 @@
-# kafka-connect-datagen
+# Overview
 
-`kafka-connect-datagen` is a Kafka Connect connector for generating mock data, not suitable for production
+`kafka-connect-datagen` is a Kafka Connect connector for generating mock data.
+It is available in [Confluent Hub](https://www.confluent.io/connector/kafka-connect-datagen/).
+It is not suitable for production.
 
-## Local
+# Confluent Platform running on local install
+
+## Install connector from Confluent Hub
+
+You may install the `kafka-connect-datagen` connector from [Confluent Hub](https://www.confluent.io/connector/kafka-connect-datagen/).
+
+```bash
+confluent-hub install --no-prompt confluentinc/kafka-connect-datagen:0.1.0
```

Comment:
Is the `--no-prompt` flag necessary here?

Reply:
No, removed for now.